roles/eda/templates/eda-api.configmap.yaml.j2

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: '{{ ansible_operator_meta.name }}-nginx-api-configmap'
+  namespace: '{{ ansible_operator_meta.namespace }}'
+data:
+  nginx_api_default_conf_template: |
+    events {
+        worker_connections 1024;
+    }
+    http {
+        map $http_x_forwarded_proto $remote_scheme {
+            default $http_x_forwarded_proto;
+            ''      $scheme;
+        }
+
+        include mime.types;
+        types {
+            application/manifest+json webmanifest;
+        }
+        server {
+            listen {{ api_nginx_port }};
+            location ^~ /static/ {
+                alias {{ static_path }}/;
```

Comment:
I did not hack around nginx conf for a while, but I was expecting to find a `root` statement here pointing to the filesystem location of the static assets

Reply:
The root directive is not strictly necessary in this configuration because the alias directive is being used in the `/static/` location block. The alias directive allows Nginx to map requests for `/static/` to a different directory on the filesystem, specified by `{{ static_path }}/`.

So I think we're good.